www/front_src/src/Header/userMenu/index.cypress.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import { mount } from '../../../../../cypress/support';
+
+import UserMenu from './index';
+
+describe('User Menu', () => {
+  beforeEach(() => {
+    cy.fixture('userMenu').as('user');
+    cy.server();
+    cy.route('GET', '**/internal.php?**', '@user').as('getUser');
```

Comment:
`route` is deprecated and will removed. You can use `intercept` instead

Reply:
I think it's a bug related to cypress, because with intercept it doesn't work !  https://github.com/cypress-io/cypress/issues/2700